### DISCUSSION TITLE
Adding a new synchronisation mode

### DISCUSSION THREAD

**Discussion by gitRonang** [vzhd1701/gridplayer#178] [Q&A]
Hello,
First of all, thank's for your great work.

I'd like to add a synchronization option.
When I add several videos (let's say 2), I want to display only the overlapping part. I already have a script that, based on the timestamp, finds the largest common sub-video and trims the original ones to match the sub-sequence. The videos are then saved in a temporary directory.

I've got a Qt window to perform the operation, and I'd like to launch GridPlayer with the video paths directly as parameters, so I don't have to drag and drop them with gridplayer.

I can't find a function or command to do this. Could you tell me if it's possible?

Thanks in advance,

Ronan

**Comment by vzhd1701** [comment]
I am not sure I understand you. GridPlayer does support sync based on percent and timestamp.

Do you want to sync all videos to the fragment of one video?
